### ISSUE TITLE
Uranium Issue Trying to Use Carbon-UI Again

### ISSUE THREAD

**Issue by mattferrin** [tuckerconnelly/uranium#5]
I think it would be nice to have the most simple radium-ui starter kit possible, and initialize it like so:

```
react-native-universal init AwesomeProject
cd AwesomeProject
react-native run-ios
```

I can't ever get it up and running from scratch without difficulty.

![simulator screen shot jan 8 2017 3 26 06 pm](https://cloud.githubusercontent.com/assets/2746787/21754129/154828fa-d5b7-11e6-9860-48778634e821.png)

**Comment by tuckerconnelly** [comment]
To fix that install react-dom@15.4.2. See https://github.com/tuckerconnelly/carbon-ui/issues/16

You can also switch off of master and on to the 0.2 and 0.1 branches for something more stable:

```
npm -S i tuckerconnelly/uranium#0.2
npm -S i tuckerconnelly/carbon-ui#0.1
```

Currently too busy to put together a boilerplate. I'll keep it in mind though.

**Comment by mattferrin** [comment]
Thank you. It worked. On to the normal font in Xcode thing.
